Mathlib/NumberTheory/FLT/Four.lean

```diff
@@ -311,3 +311,16 @@ theorem fermatLastTheoremFour : FermatLastTheoremFor 4 := by
   apply @not_fermat_42 _ _ (c ^ 2) ha hb
   rw [heq]; ring
 #align not_fermat_4 fermatLastTheoremFour
+
+/--
+To prove Fermat's Last Theorem, it suffices to prove it for odd prime exponents, and the case of
+exponent 4 proved above.
+-/
+theorem FermatLastTheorem.of_odd_primes
+    (hprimes : ∀ p : ℕ, Nat.Prime p → Odd p → FermatLastTheoremFor p) : FermatLastTheorem := by
+  intro n h
+  rw [ge_iff_le, Nat.succ_le_iff] at h
```

Comment:
Would it make sense to change the definition of FLT to `2 < n`?

Reply:
Looking at the library it seems we use `odd` (for example [Nat.Prime.eq_two_or_odd'](https://leanprover-community.github.io/mathlib4_docs/Mathlib/Data/Nat/Prime.html#Nat.Prime.eq_two_or_odd')), so I guess it makes sense to use it also here.

Reply:
Sorry, odd is totally fine. I was meaning something like this: #7869